### ISSUE TITLE
How can I create rules that include fixed classnames?

### ISSUE THREAD

**Issue by Spiralis** [nathantreid/meteor-css-modules#49]
I use font-awesome and I want to make a rule so that the font-awesome element that is a child of another CSS-nulde-based class is to have specific css-rules. 

When using font-awesome you typically add the `fa` class to the element (normally in an `<i>` tag). 

```
.myButton {
    /* any css */
}

.myButton .fa {
    /* any css */
}
```

Problem is, when I do this, CSS-modules creates a pre-fixed name for the `fa` class too. The html element has to use the verbatim fa value in order for font-awesome to work
How can I tell CSS-modules to not expand and prefix the `fa` classname?

**Comment by Spiralis** [comment]
I found the answer to this before posting the question. But, I figured that there could be others with the same question. 

This project is based on https://github.com/css-modules/css-modules, although perhaps a litt too subtle referenced? 

In that project the additional :global and :local "switches" where documented.

My code above could then be:

```
.myButton {
    /* any css */
}

.myButton :global(.fa) {
    /* any css */
}
```

or

```
.myButton {
    /* any css */
}

.myButton :global .fa {
    /* any css */
}
```
